openlibrary/macros/CreateList.html

```diff
@@ -0,0 +1,41 @@
+<div class="widget-add">
+    <input type="hidden" name="user-key" value="/people/openlibrary">
+    <input id="default-key" type="hidden" name="default-key" value="">
+    <p class="create">
+        <a href="javascript:;" class="create-new-list">Create a list</a>
+    </p>
+</div>
+<div class="hidden">
+    <div class="floaterAdd" id="addList">
+        <div class="floaterHead">
+            <h2>Create a new list</h2>
+            <a class="dialog--close" href="javascript:;">×<span class="shift">Close</span></a>
+        </div>
+        <form method="post" class="floatform" name="new-list" id="new-list">
+            <div class="formElement">
+                <div class="label">
+                    <label for="list_label">Name:</label>
+                </div>
+                <div class="input">
+                    <input type="text" name="list_label" id="list_label" class="text required" value="" required="">
+                </div>
+            </div>
+            <div class="formElement">
+                <div class="label">
+                    <label for="list_desc">Description:</label>
+                </div>
+                <div class="input">
+                    <textarea name="list_desc" id="list_desc" rows="5" cols="30"></textarea>
+                </div>
+            </div>
+            <div class="formElement">
+                <div class="input">
+                    <button id="create-list-button" type="submit" class="larger">Create new list</button>
+                    &nbsp; &nbsp;
+                    <a class="small dialog--close plain red" href="javascript:;">Cancel</a>
+                </div>
+                <div class="label"></div>
+            </div>
+        </form>
+    </div>
+</div>
```

Comment:
This is identical to the modal as it appears from the drop down ; let's pull it out into its own template file to avoid duplicating this code.

Reply:
I separated this out to be used by ReadingLogDropper and sidebar